src/alire/os_linux/alire-platform.adb

```diff
@@ -42,8 +42,15 @@ package body Alire.Platform is
                                       return Alire.Platforms.Distributions is
 
                use GNAT.Regpat;
+
+               --  Regexp accepting lines not starting with '#' like:
+               --  key=value
+               --  key='value'
+               --  key='value1 value2'
+               --  key="value"
+               --  key="value1 value2"
                Regexp : constant Pattern_Matcher :=
-                 Compile (Key & "\s*=\s*""?([^""]+)""?");
+                 Compile ("^" & Key & "=[""']?([^""']+)[""']?");
```

Comment:
This means only the first item in the list will be detected?

Reply:
There's a loop below over the items in this list, IIUC.

Reply:
This RegExp is used to find the value (being it a single string or several items separated by space characters) in lines like:
- key=value(s)
- key='value(s)'
- key="value(s)"

Depending on Multiple_Values paramater value, single of multiple values are converted into a Vector of strings.

I made some informal tests and following examples found in different os-release files are correctly processed:
- ID=ubuntu (identified as Ubuntu)
- ID=debian (identified as Ubuntu)
- ID=manjaro (correctly identified as Unknown distribution)
- ID_LIKE=arch (identified as Arch)
- ID="centos" (correctly identified as Unknown distribution)
- ID="opensuse-tumbleweed" (correctly identified as Unknown distribution)
- ID_LIKE="opensuse suse" (correctly tries opensuse then suse => identified as Unknown distribution)
- ID_LIKE="rhel fedora" (correctly tries rhel then fedora => identified as Unknown distribution)